lightly/loss/macl_loss.py

```diff
@@ -0,0 +1,58 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class MACLLoss(nn.Module):
+    def __init__(self, temperature=0.1, alpha=0.5, A_0=0):
+        super().__init__()
+        self.t_0 = temperature
+        self.alpha = alpha
+        self.A_0 = A_0
```

Comment:
```suggestion
    def __init__(self, temperature: float = 0.1, alpha: float = 0.5, A_0: float = 0):
        super().__init__()
        self.temperature = temperature
        self.alpha = alpha
        self.A_0 = A_0
```

Could you also add a short docstring that describes what the loss does, what the parameters are, and add a reference to the paper that introduced the loss? You can follow the example here: https://github.com/lightly-ai/lightly/blob/356ae56fd8f6d6bbc548d4d189d9444f780a62ca/lightly/loss/ntx_ent_loss.py#L17

Reply:
ok i will add the loss doc also